### PR TITLE
ZO-63 (alt): Less hacky way of having alternate listen directives

### PR DESCRIPTION
**Pull request by mgcm**
**:heavy_check_mark: Checklist**

- [X] A changeset describing the change and affected packages ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#changelog-and-versioning)).
- [X] Added or updated documentation.
- [ ] Tests for new functionality and regression tests for bug fixes.
- [ ] Screenshots or videos attached (for UI changes).
- [X] All your commits have a `Signed-off-by` line in the message ([more info](https://github.com/nordeck/.github/blob/main/docs/CONTRIBUTING.md#dco)).

### CODE REVIEW

**Review by HarHarLinks — CHANGES_REQUESTED** [review]
test pending, but it looks good overall, just some docs cleanup.

let's discuss which one to take provided my testing is successful just now.

**Comment by HarHarLinks** [comment]
For the results from testing, see https://github.com/nordeck/matrix-widget-toolkit/pull/860#issuecomment-2503742213

File: containers/widget-server/README.md
[the remark this answers is not in the document]
Comment (mgcm) [reply containers/widget-server/README.md:1]: Added a mention to the README as suggested.

[the remark this answers is not in the document]
Comment (HarHarLinks) [reply containers/widget-server/README.md:1]: is now NEO-1092

**Review by HarHarLinks — APPROVED** [review]

**Pull request merged**